src/java/picard/analysis/directed/HsMetrics.java

```diff
@@ -55,16 +55,16 @@
     /** The number of bases in the reference genome used for alignment. */
     public long GENOME_SIZE;
 
-    /** The number of bases which have one or more baits on top of them. */
+    /** The number of bases which are localized to one or more baits. */
     public long BAIT_TERRITORY;
 
-    /** The unique number of target bases in the experiment where target is usually exons etc. */
+    /** The unique number of target bases in the experiment where the target sequence is usually exons etc. */
```

Comment:
this is still confusing. perhaps put the where clause in paranthesis, or put a comma.